libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/coordinator/InfoPanelCoordinator.kt

```diff
@@ -139,9 +140,23 @@ internal class InfoPanelCoordinator(
         state = BottomSheetBehavior.STATE_HIDDEN
     }
 
-    private fun setGuidelinePosition(bottomSheet: View, systemBarsInsets: Insets) {
-        val parentHeight = (bottomSheet.parent as ViewGroup).height
-        guidelineBottom.setGuidelineEnd(parentHeight - bottomSheet.top - systemBarsInsets.bottom)
+    private fun updateGuidelinePosition(systemBarsInsets: Insets = context.systemBarsInsets.value) {
+        val parentHeight = (infoPanel.parent as ViewGroup).height
+        val maxPos = (parentHeight * GUIDELINE_MAX_POSITION_PERCENT).toInt()
+        val pos = parentHeight - infoPanel.top - systemBarsInsets.bottom
+        guidelineBottom.setGuidelineEnd(pos.coerceIn(0, maxPos))
+    }
+
+    private fun resetSlideOffset(bottomSheetState: Int) {
+        when (bottomSheetState) {
+            BottomSheetBehavior.STATE_EXPANDED -> 1.0f
+            BottomSheetBehavior.STATE_HALF_EXPANDED -> 0.5f
+            BottomSheetBehavior.STATE_COLLAPSED -> 0.0f
+            BottomSheetBehavior.STATE_HIDDEN -> -1.0f
```

Comment:
Let us assign these numbers to constants.

Reply:
I don't think it's necessary to assign those to constants. They are specific to BottomSheet offset calculation.

Reply:
It increases readability of the code. Initially looking at it, I was not sure what exactly what these values are why were they assigned in such a fashion.

Reply:
I see. I'll add a comment, to make it more clear.